libs/orchestrator/json_models.go

```diff
@@ -31,6 +32,7 @@ type JobJson struct {
 	RunEnvVars              map[string]string `json:"runEnvVars"`
 	StateEnvVars            map[string]string `json:"stateEnvVars"`
 	CommandEnvVars          map[string]string `json:"commandEnvVars"`
+	Region                  string            `json:"region"`
```

Comment:
It is used here: 
https://github.com/diggerhq/digger/blob/0ecc420cf9bfb736cf8dc02f9bd8d1a431f7da27/libs/orchestrator/json_models.go#L83

Because the signature of the GetProviderFromRole changed and now requires region.  

I thought this would be simplest solution since it is unlikely to expand on other configuration options at least for the forseeable future.

Reply:
oh my bad 🤦 the github diff made me think its Region added inside of `StepJson` not JobJson. This makes sense! Could we just rename it to be more explicitly `AwsRoleRegion` ? In future maybe we will refactor it into its own `AwsAssumeRoleCreds` or something like that and just calling it `Region` could lead to confusion later